lib/lz4frame.c

```diff
@@ -588,6 +588,16 @@ static void LZ4F_initStream(void* ctx,
     }
 }
 
+static int ctxTypeID_to_size(int ctxTypeID) {
```

Comment:
tip: prefer using an `enum` type for such a parameter.
If named properly, the `enum` is much preferable for readability, hence for the maintenance of the source code.